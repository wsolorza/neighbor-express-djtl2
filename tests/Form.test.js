import { shallow } from "enzyme";
import Form from "../components/Form.jsx";

let wrapper;

const block = {
  title: 'Test title',
  body: 'abc123airtableformcode'
};

beforeEach(() => {
  wrapper = shallow( <Form block={block} />);
});

describe("Form block", () => {
  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});