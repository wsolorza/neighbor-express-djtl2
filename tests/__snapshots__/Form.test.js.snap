// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form block renders correctly 1`] = `
<Content
  block={
    Object {
      "body": "abc123airtableformcode",
      "title": "Test title",
    }
  }
>
  <div
    className="grid-row grid-gap"
  >
    <div
      className="tablet:grid-col-12 usa-prose"
    >
      <h2
        className="font-heading-xl margin-top-0 tablet:margin-bottom-0"
      >
        Test title
      </h2>
      <section
        id="request_form"
      >
        <iframe
          className="airtable-embed airtable-dynamic-height"
          frameBorder="0"
          height="2316"
          onmousewheel=""
          src="https://airtable.com/embed/abc123airtableformcode"
          width="100%"
        />
      </section>
    </div>
  </div>
</Content>
`;
